intern/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+from django.utils import timezone
+
+from core.models import User, Semester, Card
+from core.utils import get_semester
+from members.models import Member
+
+
+# Create your models here.
+class AccessLevel(models.Model):
+    name = models.CharField(max_length=20)
+    uio_name = models.CharField(max_length=50)
+    description = models.TextField()
+
+    def __str__(self):
+        return self.name
+
+
+class InternGroup(models.Model):
+    name = models.CharField(max_length=50)
+    leader = models.ForeignKey(User)
+    description = models.CharField(max_length=300, null=True, blank=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Role(models.Model):
+    name = models.CharField(max_length=50)
```

Comment:
Should this be unique? Also the same applies for the other models
https://docs.djangoproject.com/en/1.9/ref/models/fields/#unique